src/app/editor/components/node-tags-bar/node-tags-bar.component.ts

```diff
@@ -66,7 +65,21 @@ export class NodeTagsBarComponent implements OnChanges {
         this.dropDown.resize();
     }
 
-    onTagSelected(tag: Tag) {
+    displayAllTags(): void {
```

Comment:
Check if this is used. No apparent call site for this method. Also see if you could just incorporate the functionality into the this.filterTags() method - returning the full list if the filterTerm is an empty string.